frontend/components/common/other/AnimatedArrow.vue

```diff
@@ -15,11 +15,14 @@ export default Vue.extend({
     props: {
         direction: {
             type: String as Vue.PropType<ArrowDirections>,
-            required: true,
+            default: undefined,
         },
```

Comment:
If no direction is provided, this component doesn't make sense anymore, right? With this approach all props will be optional. Instead, required props are a good way to spot a problem during development and check (via our storybook test) if when this component was used correctly.

Actionable here: undo modifications to the component, add `v-if` to the parent instead